metaci/repository/views.py

```diff
@@ -216,6 +219,11 @@ def create_builds(event, payload, repo, branch, release):
     for pr in repo.planrepository_set.should_run().filter(
         plan__trigger__in=["commit", "tag", "status"]
     ):
+        if not pr.plan.regex:
+            logger.warn(
```

Comment:
```suggestion
            logger.warning(
```

(warn works but is deprecated)